structs.go

```diff
@@ -835,7 +835,7 @@ type Relationship struct {
 type TooManyRequests struct {
 	Bucket     string        `json:"bucket"`
 	Message    string        `json:"message"`
-	RetryAfter time.Duration `json:"retry_after"`
+	RetryAfter float64       `json:"retry_after"`
 }
```

Comment:
This needs to be `gofmt`ed:

```suggestion
type TooManyRequests struct {
	Bucket     string  `json:"bucket"`
	Message    string  `json:"message"`
	RetryAfter float64 `json:"retry_after"`
}
```